### ISSUE TITLE
Applying outside of review

### ISSUE THREAD

**Issue by galantra** [brownbat/autoEaseFactor#7]
Taken from [here](https://github.com/eshapard/experimentalCardEaseFactor/issues/8):

> I have this installed, but I do most of my reviews on mobile (so this addon does not apply).

> Is is possible to adjust the ease factor for all cards, outside of the review period? Perhaps with a menu option to run through the decks.

**Comment by brownbat** [comment]
Thanks for bringing this over. Direct database manipulation takes me some places I haven't poked around yet, and could be risky, so this will take a while to make sure I am 100% sure I can do it safely. Would love to figure this out long term though.

**Comment by brownbat** [comment]
Implemented, you can find this in the individual deck options menu in the latest update. I know that could be a little extra work if you have a ton of decks, but doing it across a whole collection could be really slow, so I think this is safer for now.

Let me know how it goes!

**Comment by galantra** [comment]
Hi brownbat, pleased to hear the news! From the github changelog I infer that this feature is not available on versions below 2.1.26 – I guess this is necessary because of some changes in the Anki code.

I usually use 2.1.22 (because some add-ons haven't been updated yet). So, I will try the new function after I've downloaded 2.1.26.

**Comment by brownbat** [comment]
Yup, unfortunately 2.1.22 handles some of the functions differently.

I know the popular MIA add-ons for example are currently locked to 2.1.22, so I'm investigating how hard it would be to support that version too. (MIA's 6 month cycle should just put them a few weeks away from an update, if those are the ones you're waiting on they might beat me to it).

**Comment by brownbat** [comment]
2.1.22 is probably out of reach unfortunately.

[Migaku](https://github.com/migaku-official/Migaku-Japanese-Addon) began development on 2.1.35, though, so hopefully that will help all the users that want to use both out there.

**Comment by galantra** [comment]
I've updated to 2.1.35 and I don't see any additional options. I deleted an reinstalled the add-on. Still nothing.

I don't see indications that the function has been removed from the add-on. So probably sth. is wrong on my local version. Maybe interference with another add-on.

Cross-reference to https://github.com/brownbat/autoEaseFactor/issues/19

**Comment by brownbat** [comment]
Hmm, over in the deck options dropdown? Not there?

![image](https://user-images.githubusercontent.com/26754/109401283-c28c8900-791b-11eb-96aa-2e43efeacdea.png)

**Comment by aleksejrs** [comment]
It's commented out:

https://github.com/brownbat/autoEaseFactor/blob/4e965a3b76829bfd61aae63d8b59092ccf27760a/menu_action.py#L13
https://github.com/brownbat/autoEaseFactor/blob/4e965a3b76829bfd61aae63d8b59092ccf27760a/menu_action.py#L14
https://github.com/brownbat/autoEaseFactor/blob/4e965a3b76829bfd61aae63d8b59092ccf27760a/menu_action.py#L15

**Comment by brownbat** [comment]
That's just for the simulator I haven't fully integrated, the additional deck options menu option didn't face the same issue.

Instantiating a GUI toolkit that works both inside and outside Anki was my stumbling block for the Simulator, I didn't face that same issue for the "tune to performance" function. Its menu code is way over in... checks notes...

https://github.com/brownbat/autoEaseFactor/blob/4e965a3b76829bfd61aae63d8b59092ccf27760a/deck_settings.py#L106

**Comment by galantra** [comment]
> 
> 
> Hmm, over in the deck options dropdown? Not there?
> 
> ![image](https://user-images.githubusercontent.com/26754/109401283-c28c8900-791b-11eb-96aa-2e43efeacdea.png)

Hi, please excuse the late answer.

Nope, no options for me as of Version ⁨2.1.40 (cf446733)⁩.

I haven't found out why that is the case.

Personally, I currently do all my reviews on desktop. So, I don't mind that the options don't appear. As far as I'm concerned, the issue requires no further action.
